config/schemas/config.yaml

```
@@ -127,6 +371,1362 @@ properties:
+                  type: object
+                type: array
+              space:
+                items:
+                  additionalProperties: false
+                  properties:
+                    for:
+                      default: 30m
+                      type: string
+                    freeSpacePercentage:
+                      $ref: '#/$defs/percentage'
+                      default: 5
+                    pattern:
+                      additionalProperties: false
+                      properties:
+                        exclude:
+                          additionalProperties: false
+                          properties:
+                            disk:
+                              default: ""
+                              type: string
+                            node:
+                              default: ""
+                              type: string
+                          type: object
+                        include:
+                          additionalProperties: false
+                          properties:
+                            disk:
+                              default: .*
+                              type: string
+                            node:
+                              default: .*
+                              type: string
+                          type: object
+                      type: object
+                    severity:
+                      default: warning
+                      type: string
+                  type: object
+                type: array
+            type: object
+        type: object
+      nodeSelector:
+        $ref: '#/$defs/kubernetesNodeSelector'
+      replicas:
+        default: 1
+        type: number
+      resources:
+        $ref: '#/$defs/kubernetesResourceRequirements'
+      retention:
+        additionalProperties: false
+        properties:
+          age:
+            default: 3d
+            type: string
+          alertmanager:
+            $ref: '#/$defs/timeRange'
+            default: 72h
+          size:
+            default: 4GiB
+            type: string
+        type: object
+      s3BucketAlerts:
+        additionalProperties: false
+        properties:
+          exclude:
+            type: array
+          objects:
+            additionalProperties: false
+            properties:
+              count:
+                default: 1638400
+                type: number
+              enabled:
+                default: false
+                type: boolean
+              percent:
+                $ref: '#/$defs/percentage'
+                default: 80
+            type: object
+          size:
+            additionalProperties: false
+            properties:
+              enabled:
+                default: false
+                type: boolean
+              percent:
+                $ref: '#/$defs/percentage'
+                default: 80
+              sizeQuotaGB:
+                default: 1000
+                type: number
```

`totalSize` was recently added by me, I did not think about adding it to the schema so would be nice if you could add it here:
```suggestion
                type: number
          totalSize:
            additionalProperties: false
            properties:
              enabled:
                default: false
                type: boolean
              percent:
                $ref: '#/$defs/percentage'
                default: 80
              sizeQuotaGB:
                default: 1000
                type: number
            type: object
```